sdk/core/azure-core-http-vertx/README.md

```diff
@@ -0,0 +1,85 @@
+# Azure Core Vert.x HTTP plugin library for Java
+
+Azure Core Vert.x HTTP client is a plugin for the `azure-core` HTTP client API.
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+
+### Include the package
+
+```xml
```

Comment:
```suggestion
[//]: # ({x-version-update-start;com.azure:azure-core-http-vertx;current})
```xml
```

Reply:
I've now added this markup to the README.